### PR TITLE
LPS-147645 User cannot configure priority with negative values ​​within actions on a node

### PR DESCRIPTION
**Pull request by kildarep**

### CODE REVIEW

**Comment by kildarep** [comment]
ci:test:sf

**Comment by kildarep** [comment]
ci:test:relevant

**Comment by cgmonte** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-const */
```
Comment (cgmonte) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js:1]: Beware of disabling eslint rules for the whole file. These rules are important and should be followed by developers. In cases where you think a rule should not be applied, please add eslint disable comments for specific lines or blocks of code. Also, the license comment statement always comes at the top.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js [hunk 2]
```diff
@@ -199,16 +197,8 @@ const BaseActionsInfo = ({
 				<ClayInput
 					aria-label="Select"
 					id="priority"
-					min={MIN_PRIORITY}
 					onBlur={({target}) => {
 						let {value: newValue} = target;
```
Comment (cgmonte) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js:201]: change to `const.`

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js [hunk 3]
```diff
@@ -221,10 +211,6 @@ const BaseActionsInfo = ({
 					}}
 					onChange={({target}) => {
 						let {value: newValue} = target;
```
Comment (cgmonte) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js:213]: change to `const`

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi @kildarep! Please, read the comments I made on your commit. By the way, commit messages should be a description of the changes made by the commit, not the the summary of the ticket. Always start with LPS-XXXXXX, but then describe the commit changes. (i.e., LPS-147645 Remove minimum value checking from priority field in actions.)

**Comment by kildarep** [comment]
ci:test:sf

**Comment by kildarep** [comment]
ci:test:sf

**Comment by kildarep** [comment]
ci:test:relevant

**Comment by kildarep** [comment]
ci:test:relevant

**Review by cgmonte — APPROVED** [review]

**Comment by cgmonte** [comment]
ci:forward

**Pull request closed without merging**